cmd/diff.go

```diff
@@ -30,59 +36,62 @@ func newDiffCmd() *cobra.Command {
 				It is used on two files of the same format from any format
 				supported by the asconfig tool, e.g. yaml or Aerospike config.
 				Schema validation is not performed on either file. The file names must end with
-				extensions signifying their formats, e.g. .conf or .yaml, or --format must be used.`,
+				extensions signifying their formats, e.g. .conf or .yaml, or --format must be used.
+				The first argument may be stdin.`,
```

Comment:
We talked about removing this with @hev. If not this like should be reworded like what was done above

Reply:
reworded to `If only one file path is provided diff reads the other file contents from stdin.` for now.